src/box/vy_run.c

```diff
@@ -2558,6 +2580,14 @@ vy_run_remove_files(const char *dir, uint32_t space_id,
 		} else
 			say_info("removed %s", path);
 	}
+
+	/* Remove the root directory if it's empty. */
+	vy_lsm_snprint_path(path, sizeof(path), dir, space_id, iid);
+	if (try_rmdir(path) < 0)
+		return -1;
+	snprintf(path, sizeof(path), "%s/%u", dir, space_id);
```

Comment:
Please add a helper function `vy_space_snprint_path()` and make `vy_lsm_snprint_path()` use it so that path formatting remains in one place.

Reply:
Fixed.